src/sqs/push.ts

```diff
@@ -22,22 +22,20 @@ export type PushQueueMessagePayload = {
 	webhookReceived?: Date,
 }
 
-export const pushQueueMessageHandler : MessageHandler<PushQueueMessagePayload> = {
+export const pushQueueMessageHandler : MessageHandler<PushQueueMessagePayload> = async (context : Context<PushQueueMessagePayload>) => {
```

Comment:
I had to address some PR comments and made this as a function instead of an object. No other changes in this file